spec/swagger.yaml

```diff
@@ -8050,4 +8106,470 @@ definitions:
+                    enum:
+                      - OPENED
+                      - RESOLVED
+  AccountIssueTypesResponse:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
+            totalCount:
+              type: integer
+            items:
+              type: array
+              items:
+                type: object
+                properties:
+                  description:
+                    type: string
+                  issueTypeCode:
+                    type: string
+                    enum:
+                      - SOURCE
+                      - TRANSLATION
+  CreateIssueResponse:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
+            string:
+              type: object
+              properties:
+                hashcode:
+                  type: string
+                localeId:
+                  type: string
+            resolvedByUserUid:
+              type: string
+            reportedByUserUid:
+              type: string
+            createdDate:
+              type: string
+            resolvedDate:
+              type: string
+            issueText:
+              type: string
+            issueUid:
+              type: string
+            projectId:
+              type: string
+            issueTypeCode:
+              type: string
+              enum:
+                - SOURCE
+                - TRANSLATION
+            issueSubTypeCode:
+              type: string
+              enum:
+                - CLARIFICATION
+                - MISSPELLING
+                - POOR_TRANSLATION
+                - DOES_NOT_FIT_SPACE
+                - PLACEHOLDER_ISSUE
+                - GENERAL_ISSUE
+                - REVIEW_TRANSLATION
+                - CUSTOM
+            issueStateCode:
+              type: string
+              enum:
+                - OPENED
+                - RESOLVED
+            issueTextModifiedDate:
+              type: string
+            answered:
+              type: boolean
+  IssueTextResponse:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
+            issueText:
+              type: string
+            issueTextModifiedDate:
+              type: string
+  IssueStateResponse:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
+            issueStateCode:
+              type: string
+              enum:
+                - OPENED
+                - RESOLVED
+  IssueGetResponse:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
```

Comment:
Is possible to extract issue and watcher response properties into a separate item and to reuse it wherever needed? Like for get/find/create issues responses?
See https://github.com/Smartling/api-docs/blob/815634552908f52e001413ec930c452dd7895e7c/spec/swagger.yaml#L7777 as an example